packages/react-core/src/components/Tabs/__tests__/OverflowTab.test.tsx

```diff
@@ -376,7 +377,7 @@ test('Passes Popper an appendTo value of the presentation element', async () =>
   await user.click(screen.getByRole('tab'));
 
   // This assertion relies on the structure of the Popper mock to verify the correct props are being sent to Popper
-  expect(screen.getByText('Append to class name: pf-v5-c-tabs__item pf-m-overflow')).toBeVisible();
+  expect(screen.getByText(`Append to class name: ${styles.tabsItem} pf-m-overflow`)).toBeVisible();
```

Comment:
should this use the `styles.modifiers` object?

```suggestion
  expect(screen.getByText(`Append to class name: ${styles.tabsItem} ${styles.modifiers.overflow}`)).toBeVisible();
```

Reply:
Cool, sounds good! There could be a future where we version modifiers, and if we used `styles.modifiers`, it would pull that in. IMO it's probably better to not hard code as much as we can, too.